### PR TITLE
Doc fix qceff-examples & netcdf inflation files

### LINKED ISSUE #710: doc: out of date QCEFF examples with the lorenz tracer model
https://docs.dart.ucar.edu/en/latest/guide/qceff-examples.html

Should be removed:

To get started, make sure that you are on the quantile_methods branch of DART:

   git checkout quantile_methods

@hkershaw put in next release.

### PR DESCRIPTION
**Pull request by hkershaw-brown**
## Description:

Removes out-of-date reference to quantile-methods branch
Removes references to non-existant netcdf_inflation_files.rst (removed in #698) 

### Fixes issue

fixes #710 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Documentation changes needed?

- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.

### Tests
Documentation build

## Checklist for merging

- [ ] Updated changelog entry
- [ ] Documentation updated
- [ ] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [ ] No dataset needed

### CODE REVIEW

**Comment by kdraeder** [comment]
Sorry that I didn't respond to this earlier.  (I was busy reading these docs and adapting reanalysis branch scripts to use qceff capabilities).

Removing refs to the quantile_methods is good.

I also wondered whether the qceff table replaces the assim_tools_nml method described in the "previous" page,
in which case that file should be updated, maybe as part of this PR.

A link to qceff_examples in qceff_probit would have been useful to me,
although I found it through a search.

In the qceff_probit.rst and the qceff-examples.rst I recommend changing "anxious" to "eager"

**Comment by hkershaw-brown** [comment]
more and more duplicate docs:

https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters

**Comment by hkershaw-brown** [comment]
and more: https://docs.dart.ucar.edu/en/latest/guide/important-capabilities-dart.html#data-assimilation-algorithms-available-in-dart

File: guide/important-capabilities-dart.rst [hunk 1]
```diff
@@ -144,10 +144,9 @@ Data assimilation algorithms available in DART
 DART allows users to test the impact of using multiple different types of
 algorithms for filtering, inflation/deflation, and covariance localization.
 
-DART offers numerous **filter algorithms**. These determine how the posterior
+DART offers numerous **filter algorithms** in the :ref:`QCEFF`. These determine how the posterior
```
Comment (kdraeder) [inline guide/important-capabilities-dart.rst:147]: I believe "posterior" should be "prior".  The prior is updated by the algorithm to yield the posterior.   And then remove "and the prior ensemble"

Comment (hkershaw-brown) [reply guide/important-capabilities-dart.rst]: Thanks for checking this Kevin. changed to your suggestion.  Better?

Comment (kdraeder) [reply guide/important-capabilities-dart.rst]: I see the fix in "Commits", but not in "Files Changed".
The fix looks good.

**Comment by kdraeder** [comment]
> more and more duplicate docs:
> 
> https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters

I was going to approve the PR, but then remembered these 2 notes about duplicates.
Are there fixes on the way that I should review?

**Comment by hkershaw-brown** [comment]
> > more and more duplicate docs:
> > https://dart-documentation--711.org.readthedocs.build/en/711/guide/filters.html#filters
> 
> I was going to approve the PR, but then remembered these 2 notes about duplicates. Are there fixes on the way that I should review?

yeah let me push em up.

**Comment by hkershaw-brown** [comment]
@kdraeder 
  removed filter.rst file (seems pointless) and removed that file from table of contents
  removed filter_kind  assim_tools_nml comment from guide/important-capabilities-dart.rst
  removed filter_kind from example assim_tools_nmls in:
     - models/clm/tutorial/README.rst
     - models/cm1/readme.rst

That was all the filter_kind assim_tools_nml references I could find.

File: models/clm/tutorial/README.rst
Comment (kdraeder) [inline models/clm/tutorial/README.rst:1]: typo; witin -> within

Comment (kdraeder) [reply models/clm/tutorial/README.rst:1]: This is not a show-stopper, and the other recent changes look fine.

Comment (hkershaw-brown) [reply models/clm/tutorial/README.rst:1]: I fixed it, then put a typo in my commit message. Conservation of typos.

**Review by kdraeder — APPROVED** [review]
Onward and sideways!  There's probably a way to edit comments, but I wouldn't bother.
This issue is wrapped up well enough for me!

**Pull request merged**